### PR TITLE
chore: Remove DesignKit since it's now maintained in Hew [WEB-1790]

### PR DESCRIPTION
**Pull request by mapmeld**
## Description

The DesignKit and visual diffs are now in Hew
For example in this PR https://github.com/determined-ai/hew/pull/27 it got an automated UI kit diff

## Test Plan

Any removed tests are running in Hew
Existing site builds, tests pass

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by ashtonG** [comment]
you should be able to remove `scripts/make-github-comment.mjs` as well

**Comment by keita-determined** [comment]
also, `scripts/make-screenshot-diff.mjs` too? 
i think we should be able to remove almost all from #6707.

**Comment by johnkim-det** [comment]
since @keita-determined identified #6707, I took a look, and I think there's more that could be removed from `.gitignore` and `Makefile`

**Comment by mapmeld** [comment]
@johnkim-det we did some research late yesterday
- undici dependency no longer used
- removing the remaining playwright reference in Makefile caused [these errors](https://app.circleci.com/pipelines/github/determined-ai/determined/47661/workflows/89218823-685b-4e9e-9f91-000d7d9db905/jobs/2030898)
- I would argue to keep `.gitignore`, if someone generated these files we still don't want them getting committed

File: webui/react/vite.config.mts [hunk 1]
```diff
@@ -141,9 +140,6 @@ export default defineConfig(({ mode }) => ({
       checker({
         typescript: true,
       }),
-    devServerRedirects({
```
Comment (ashtonG) [inline webui/react/vite.config.mts:144]: should be able to remove the `devServerRedirects` plugin here

**Review by johnkim-det — APPROVED** [review]

**Pull request merged**